.github/workflows/release.yml

```diff
@@ -0,0 +1,81 @@
+name: Release
+
+on:
+  pull_request_target:
+    types:
+      - closed
+concurrency:
+  group: tests-${{ github.ref }}
```

Comment:
We might need to change the group name here. And maybe we should not "cancel-in-progress"?

Reply:
It would be more general to use `${{ github.workflow }}-${{ github.ref }}` instead here.